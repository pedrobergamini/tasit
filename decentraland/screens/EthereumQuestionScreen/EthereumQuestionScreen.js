import React from "react";
import EthereumQuestion from "@presentational/EthereumQuestion";

export default class EthereumQuestionScreen extends React.Component {
  render() {
    return (
      <EthereumQuestion
        onSignUp={() => this.props.navigation.navigate("EthereumSignUpScreen")}
        onSignIn={() => this.props.navigation.navigate("EthereumSignInScreen")}
      />
    );
  }
}
