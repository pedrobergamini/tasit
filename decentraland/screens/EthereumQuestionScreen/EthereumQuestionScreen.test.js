import React from "react";
import NavigationTestUtils from "react-navigation/NavigationTestUtils";
import { shallow } from "enzyme";
import EthereumQuestionScreen from "./EthereumQuestionScreen";

describe("EthereumQuestionScreen", () => {
  jest.useFakeTimers();
  beforeEach(() => {
    NavigationTestUtils.resetInternalState();
  });

  it("renders the component", async () => {
    expect(shallow(<EthereumQuestionScreen />)).toMatchSnapshot();
  });
});
