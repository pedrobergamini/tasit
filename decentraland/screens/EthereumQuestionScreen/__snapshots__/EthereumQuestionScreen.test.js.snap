// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`EthereumQuestionScreen renders the component 1`] = `
<EthereumQuestion
  onSignIn={[Function]}
  onSignUp={[Function]}
/>
`;
